EngineLayer/Mods/tmt.txt

```diff
@@ -1,4 +1,65 @@
 ﻿TMT Multiplex
+ID   TMT11
+TG   X
+MT   Multiplex Label
+PP   Peptide N-terminal.
+MM   229.162932
+#NL   HCD:C8 N1 H15
+DI   HCD:C8 N1 H15
+#NL   HCD:C8 H15 N{15}1
+DI   HCD:C8 H15 N{15}1
+#NL   HCD:C7 H15 C{13}1 N{15}1
+DI   HCD:C7 H15 C{13}1 N{15}1
+#NL   HCD:C6 H15 C{13}2 N{15}1
+DI   HCD:C6 H15 C{13}2 N{15}1
+#NL   HCD:C5 H15 C{13}3 N{15}1
+DI   HCD:C5 H15 C{13}3 N{15}1
+#NL   HCD:C3 N1 H15 C{13}5
+DI   HCD:C3 N1 H15 C{13}5
+#NL   HCD:C7 N1 H15 C{13}1
+DI   HCD:C7 N1 H15 C{13}1
+#NL   HCD:C6 N1 H15 C{13}2
+DI   HCD:C6 N1 H15 C{13}2
+#NL   HCD:C5 N1 H15 C{13}3
+DI   HCD:C5 N1 H15 C{13}3
+#NL   HCD:C4 N1 H15 C{13}4
+DI   HCD:C4 N1 H15 C{13}4
+#NL   HCD:C4 H15 C{13}4 N{15}1
+DI   HCD:C4 H15 C{13}4 N{15}1
+#NL   HCD:C3 H16 N1 C{13}5
+DI   HCD:C3 H16 N1 C{13}5
+//
+ID   TMT11
+TG   K
+MT   Multiplex Label
+PP   Anywhere.
+MM   229.162932
+#NL   HCD:C8 N1 H15
+DI   HCD:C8 N1 H15
+#NL   HCD:C8 H15 N{15}1
+DI   HCD:C8 H15 N{15}1
+#NL   HCD:C7 H15 C{13}1 N{15}1
+DI   HCD:C7 H15 C{13}1 N{15}1
+#NL   HCD:C6 H15 C{13}2 N{15}1
+DI   HCD:C6 H15 C{13}2 N{15}1
+#NL   HCD:C5 H15 C{13}3 N{15}1
+DI   HCD:C5 H15 C{13}3 N{15}1
+#NL   HCD:C3 N1 H15 C{13}5
+DI   HCD:C3 N1 H15 C{13}5
+#NL   HCD:C7 N1 H15 C{13}1
+DI   HCD:C7 N1 H15 C{13}1
+#NL   HCD:C6 N1 H15 C{13}2
+DI   HCD:C6 N1 H15 C{13}2
+#NL   HCD:C5 N1 H15 C{13}3
+DI   HCD:C5 N1 H15 C{13}3
+#NL   HCD:C4 N1 H15 C{13}4
+DI   HCD:C4 N1 H15 C{13}4
+#NL   HCD:C4 H15 C{13}4 N{15}1
+DI   HCD:C4 H15 C{13}4 N{15}1
+#NL   HCD:C3 H16 N1 C{13}5
+DI   HCD:C3 H16 N1 C{13}5
```

Comment:
Did you test with real TMT data?

Reply:
![image](https://user-images.githubusercontent.com/16841846/60985444-21465e80-a303-11e9-825c-e9d7f37b452a.png)

Reply:
![image](https://user-images.githubusercontent.com/16841846/60985515-40dd8700-a303-11e9-83b9-b7b264c7483b.png)